### PR TITLE
[[ Browser ]] Ensure libbrowser functionality isn't dead stripped

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

**Comment by livecodefraser** [comment]
Is there a reason we can't use the definitions of MC_DLLEXPORT{,_DEF} from libfoundation rather than re-defining them?

File: docs/notes/bugfix-16436.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Define MC_BROWSER_DLLEXPORT(_DEF) in libbrowser.h identically to MC_DLLEXPORT(_DEF) in foundation.h
```
Comment (peter-b) [inline docs/notes/bugfix-16436.md:1]: I think you might need to quote the `_` characters to avoid Markdown weirdness.

**Comment by peter-b** [comment]
@livecode-vulcan review ok 9b61313

@livecodepanos I think this will fix the libbrowser problems.

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 9b613132b012f68c39e4e5cdc460f751a2440da6

**Comment by livecode-vulcan** [comment]
😞 test failure 9b613132b012f68c39e4e5cdc460f751a2440da6
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1719)
- try-community-mac: [pending](https://vulcan.livecode.com/builders/try-community-mac/builds/1735)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1714)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1725)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1736)
- try-community-win-x86: [failure](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1738)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1743)

**Comment by peter-b** [comment]
@livecodeali  Makes sense :+1: 

@livecode-vulcan review ok 89f2db6

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 89f2db67f932c1ef40b6be81da4572388ff62d0e

**Comment by livecode-vulcan** [comment]
😎 test success 89f2db67f932c1ef40b6be81da4572388ff62d0e
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1724)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1739)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1719)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1730)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1740)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1742)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1748)

**Pull request merged**